### ISSUE TITLE
Table Uploader Suggestions/New Features

### ISSUE THREAD

**Issue by faithtek** [Kademi/kademi-dev#6814]
When a someone uploads and there are errors, it would be nice to retain those error files somewhere, so that even after the user clicks of the page, they have the ability to go back and download that file and re-upload. This would also help me to better support my clients, as I have a few clients that upload themselves and sometimes get errors (and an error file) that I need to look at and correct and re-upload.

I don't think these error files necessarily need to be kept forever, but maybe 1 month, max 3 months.

**Comment by bradmac** [comment]
Agreed. This is intended functionality, so good to have an issue for it.

BTW, we would just keep the files forever, having an automated process to clean up would be a lot of extra work, and we have no shortage of storage space.
